source/adam.cpp

```diff
@@ -38,7 +38,7 @@
 /**************************************************************************************************/
 
 using namespace std;
-using namespace std::placeholders;
+namespace ph = std::placeholders;
```

Comment:
This is ugly. What is the ambiguity? Before accepting this change, I'd like to understand the issue.

Reply:
A quick test - https://godbolt.org/z/Ydjff99Mo
Where does the ambiguity originate?

Reply:
Error message is as following, `boost/bind.hpp` is indirectly included through `boost/signals2/signal.hpp`.
```
/root/expr/adobe_source_libraries/source/adam.cpp: In member function ‘adobe::sheet_t::connection_t adobe::sheet_t::implementation_t::monitor_enabled(adobe::name_t, const adobe::name_t*, const adobe::name_t*, const monitor_enabled_t&)’:
/root/expr/adobe_source_libraries/source/adam.cpp:939:92: error: reference to ‘_1’ is ambiguous
  939 |                                                  touch_set, iter->cell_set_pos_m, monitor, _1, _2));
      |                                                                                            ^~
In file included from /root/expr/adobe_source_libraries/source/../adobe/adam.hpp:13,
                 from /root/expr/adobe_source_libraries/source/adam.cpp:8:
/usr/include/c++/9/functional:211:34: note: candidates are: ‘const std::_Placeholder<1> std::placeholders::_1’
  211 |     extern const _Placeholder<1> _1;
      |                                  ^~
In file included from /usr/include/boost/bind/bind.hpp:2356,
                 from /usr/include/boost/bind.hpp:22,
                 from /usr/include/boost/signals2/slot.hpp:15,
                 from /usr/include/boost/signals2/connection.hpp:24,
                 from /usr/include/boost/signals2/signal.hpp:22,
                 from /root/expr/adobe_source_libraries/source/../adobe/adam.hpp:16,
                 from /root/expr/adobe_source_libraries/source/adam.cpp:8:
/usr/include/boost/bind/placeholders.hpp:46:38: note:                 ‘constexpr const boost::arg<1> boost::placeholders::_1’
   46 | BOOST_STATIC_CONSTEXPR boost::arg<1> _1;
```